diff --git a/node_modules/@mswjs/interceptors/lib/browser/chunk-PSX5J3RF.js b/node_modules/@mswjs/interceptors/lib/browser/chunk-PSX5J3RF.js
index 37e2a0f..48d16d6 100644
--- a/node_modules/@mswjs/interceptors/lib/browser/chunk-PSX5J3RF.js
+++ b/node_modules/@mswjs/interceptors/lib/browser/chunk-PSX5J3RF.js
@@ -457,6 +457,13 @@ var XMLHttpRequestController = class {
         readNextResponseBodyChunk();
       };
       readNextResponseBodyChunk();
+    } else if (response._bodyInit) {
+      this.logger.info('mocked response has _bodyInit, faking streaming...')
+
+      const bodyInit = response._bodyInit
+      const encoder = new TextEncoder()
+      this.responseBuffer = encoder.encode(bodyInit)
+      finalizeResponse()
     } else {
       finalizeResponse();
     }
