diff --git a/node_modules/expo-router/build/global-state/routing.js b/node_modules/expo-router/build/global-state/routing.js
index 2fba9d1..9cce0c9 100644
--- a/node_modules/expo-router/build/global-state/routing.js
+++ b/node_modules/expo-router/build/global-state/routing.js
@@ -173,6 +173,9 @@ function getNavigateAction(state, parentState, type = 'NAVIGATE') {
     else if (type === 'REPLACE' && parentState.type === 'tab') {
         type = 'JUMP_TO';
     }
+
+    // https://github.com/expo/expo/issues/26211
+    params.initial = false
+    
     return {
         type,
         target: parentState.key,
